contracts/DebondToken.sol

```diff
@@ -155,6 +155,20 @@ abstract contract DebondToken is IDebondToken, ERC20, ExecutableOwnable {
         // as the airdroped supply is minted it will be seperate from the each investors lockedBalance.
     }
 
+    function mintAllocatedSupply(address _to, uint256 _amount)
```

Comment:
So the only major change is the **mintAllocatedSupply** function that has been moved to DebondToken abstract contract and removed from both DEBIT and DGOV, right ?